system/Validation/Validation.php

```diff
@@ -274,7 +285,8 @@ protected function processRules(string $field, ?string $label, $value, $rules =
                     }
 
                     $found  = true;
-                    $passed = $param === false ? $set->{$rule}($value, $error) : $set->{$rule}($value, $param, $data, $error);
+                    $passed = $param === false ? $set->{$rule}($value, $error)
+                        : $set->{$rule}($value, $param, $data, $error);
```

Comment:
The `operator_linebreak` should break this one.
```suggestion
                    $passed = $param === false
                        ? $set->{$rule}($value, $error)
                        : $set->{$rule}($value, $param, $data, $error);
```